components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -494,6 +494,12 @@ initialize_fields (const util::TimeStamp& run_t0, const util::TimeStamp& case_t0
     }
   }
 
+#ifdef SCREAM_HAS_MEMORY_USAGE
+  long long my_mem_usage = get_mem_usage(MB);
+  long long max_mem_usage;
+  m_atm_comm.all_reduce(&my_mem_usage,&max_mem_usage,1,MPI_MAX);
+  m_atm_logger->info("[EAMxx::init::initialize_fields] memory usage: " + std::to_string(max_mem_usage) + "MB");
```

Comment:
These new calls to the memory usage in the middle of init sequence seems more of a "debug" log level than "info"...